### PR TITLE
Updates kubeadm default to use 1.10

### LINKED ISSUE #60608: kubeadm v1.10 release tracking issue
In following with history we will be curating action items here, but it's still a WIP, atm: https://github.com/kubernetes/kubeadm/milestone/5 

All non-bug-fixes for this release cycle should be merged, but there is still an ongoing list of bugs that folks may execute against for the remainder of the freeze and we will evaluating on a case-by-case basis.

/cc @detiber @jdumars @roberthbailey @kubernetes/kubernetes-release-managers @luxas @kubernetes/sig-cluster-lifecycle-misc

### PR DESCRIPTION
**Pull request by timothysc**
**What this PR does / why we need it**:
In line with https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md, the default branch for kubeadm to deploy should be bumped right before the rc.1.
This can even be manually merged by the someone from the release team.

**Which issue(s) this PR fixes** :
Fixes #60608 

**Special notes for your reviewer**:

**Release note**:

```
Updates default deployment for kubeadm to 1.10 
```

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews @kubernetes/sig-release-members  @dims @jberkus @jdumars

### CODE REVIEW

**Comment by timothysc** [comment]
Someone from the release team will have to switch the labels when rc.1 is ready to be cut.

**Comment by jdumars** [comment]
Can we get the LGTM so we're not scrambling at the last minute when it's time to merge?

**Review by jamiehannaford — APPROVED** [review]
/lgtm

**Review by luxas — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
